admin/acceptance.py

```diff
@@ -636,12 +635,15 @@ def _create_and_provision(self, reactor, name, retries=10):
         :return: Deferred that fires with the created node when
             the provisioning is completed.
         """
+        print "re-creating and re-trying node", name
+
         def create_attempt():
-            d = deferToThreadPool(
-                reactor,
-                reactor.getThreadPool(),
-                self._create_node,
-                name,
+            # A list with a single Deferred must be returned.
```

Comment:
I assumed it to be sort of self-documented in the code as `create_nodes` is called with a single node name.